src/wallet.js

```diff
@@ -235,17 +263,22 @@ const wallet = {
    * @param {string} pin
    * @param {string} password
    * @param {boolean} loadHistory if should load the history from the generated addresses
+   * @param {object} multisig MultiSig data including minSignatures and xpubs of participants
    *
    * @return {Promise} Promise that resolves when finishes loading address history, in case loadHistory = true, else returns null
    * @memberof Wallet
    * @inner
    */
-  executeGenerateWallet(words, passphrase, pin, password, loadHistory) {
+  executeGenerateWallet(words, passphrase, pin, password, loadHistory, multisig) {
     let code = new Mnemonic(words);
     let xpriv = code.toHDPrivateKey(passphrase, network.getNetwork());
-    let privkey = xpriv.deriveNonCompliantChild(`m/44'/${HATHOR_BIP44_CODE}'/0'/0`);
     let authXpriv = xpriv.deriveNonCompliantChild(WALLET_SERVICE_AUTH_DERIVATION_PATH);
-
+    let privkey;
+    if (multisig) {
+      privkey = xpriv.deriveNonCompliantChild(`m/45'/${HATHOR_BIP44_CODE}'/0'/0`);
```

Comment:
We should use the var from the constant file here

Reply:
Done in [89d3a89](https://github.com/HathorNetwork/hathor-wallet-lib/pull/324/commits/89d3a89fb831ed94006445e3d29178170df31c29)